.github/workflows/publish.yml

```diff
@@ -0,0 +1,27 @@
+name: Publish
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Enter a valid semver version'
+        required: true
```

Comment:
What does this do exactly?

Reply:
Soz, was just updating the OP with images. Raising PRs from the command line is convenient when you don't want to add images :P

Reply:
Nice, looks good